src/Surfnet/StepupSelfService/SamlStepupProviderBundle/DependencyInjection/SurfnetStepupSelfServiceSamlStepupProviderExtension.php

```diff
@@ -173,9 +182,9 @@ private function buildHostedEntityDefinition($provider, array $configuration, ar
         ];
 
         $serviceProvider  = array_merge($configuration['service_provider'], $spAdditional, $entityId);
-        $identityProvider = array_merge($idpAdditional, $entityId);
+        $identityProvider = [...$idpAdditional, ...$entityId];
 
-        $hostedDefinition = new Definition('Surfnet\SamlBundle\Entity\HostedEntities', [
+        $hostedDefinition = new Definition(HostedEntities::class, [
```

Comment:
How is this not triggering a multi line PHP CS issue? where if you decide to break a statement into a multiliner. Every part of the statement (after every comma) you have to add a linefeed.